### PR TITLE
[W5.8b][W11-A1]Lai Qi Wei

### PR DESCRIPTION
**Pull request by qwlai**

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should respond to all commands within two seconds under normal load.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:97]: What is normal load?

**Comment by okkhoy** [comment]
@qwlai 
NFR (and requirements in general) should be as precise as possible.
Avoid adjectives like `easy`, `fast`, `intuitive` etc.,
Instead, use precise statements like `response time should be 1 second`, `detailed help which shows all options should be present`

Some comments added. Please ack & close the PR after reading comments.

File: doc/DeveloperGuide.md (hunk 1 above)
Comment (qwlai) [reply doc/DeveloperGuide.md:97]: I should have been clearer with this term. Normal load in this case means cpu load but I see what you mean, normal load is ambiguous.

**Comment by qwlai** [comment]
Noted with thanks.

**Pull request closed without merging**